site/docs/get-started/vagrant.md

```diff
@@ -19,10 +19,10 @@ Additional details on how to configure the system properly can be found in the l
 
 ### Step 2. Download and deploy the appliance
 
-1. Pull the ManageIQ Vagrant box from [Atlas](https://atlas.hashicorp.com/vagrant) and run it
+1. Pull the ManageIQ Vagrant box from [Atlas](https://atlas.hashicorp.com/ManageIQ) and run it
```

Comment:
Also convention seems to be to use lower case on Atlas. That would be easier to type.